src/sentry/static/sentry/app/components/events/contextSummary.jsx

```diff
@@ -273,15 +282,15 @@ class EventContextSummary extends React.Component {
     if (contextCount < MIN_CONTEXTS) {
       // Add contents in the declared order until we have at least MIN_CONTEXTS
       // contexts in our list.
-      contexts = KNOWN_CONTEXTS.map(({key, Component, ...props}, index) => {
+      contexts = KNOWN_CONTEXTS.map(({keys, Component, ...props}, index) => {
         if (contexts[index]) {
           return contexts[index];
         }
         if (contextCount >= MIN_CONTEXTS) {
           return null;
         }
         contextCount += 1;
-        return <Component key={key} data={{}} {...props} />;
+        return <Component key={keys[0]} data={{}} {...props} />;
```

Comment:
Shouldn't this also iterate through all keys, i.e. flatten `keys` out?

Reply:
If we do this then we will attach two empty os contexts